pkgs/development/python-modules/pysonos/default.nix

```diff
@@ -7,7 +7,7 @@
 , ifaddr
 
 # Test dependencies
-, pytest, pylint, flake8, graphviz
+, pytestCheckHook, pylint, flake8, graphviz
 , mock, sphinx, sphinx_rtd_theme
 , requests-mock
```

Comment:
Please clean up the test inputs. Most of them (pylint, flake8, graphviz, sphinx) are unnecessary.

Also, please migrate to one argument per line.

Reply:
Thanks, done